diff --git a/node_modules/autoprefixer/lib/processor.js b/node_modules/autoprefixer/lib/processor.js
index de13fb7..1384909 100644
--- a/node_modules/autoprefixer/lib/processor.js
+++ b/node_modules/autoprefixer/lib/processor.js
@@ -113,11 +113,6 @@ class Processor {
       let value = decl.value
 
       if (prop === 'color-adjust') {
-        result.warn(
-          'Replace color-adjust to print-color-adjust. ' +
-            'The color-adjust shorthand is currently deprecated.',
-          { node: decl }
-        )
       } else if (prop === 'grid-row-span') {
         result.warn(
           'grid-row-span is not part of final Grid Layout. Use grid-row.',
